### PR TITLE
[Feature cleanup] remove cc for hide attendee names

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Remove CC lines for hide attendee names
# Why

code cleanup

https://skype.visualstudio.com/SPOOL/_workitems/edit/3813487
# How Tested

Built locally CI to test as well

### CODE REVIEW

File: packages/calling-component-bindings/src/participantListSelector.ts [hunk 1]
```diff
@@ -164,8 +159,8 @@ export const participantListSelector: ParticipantListSelector = createSelector(
       ? convertRemoteParticipantsToParticipantListParticipants(
           updateUserDisplayNamesTrampoline(Object.values(remoteParticipants)),
           localUserCanRemoveOthers,
-          undefined || /* @conditional-compile-remove(hide-attendee-name) */ isHideAttendeeNamesEnabled,
-          undefined || /* @conditional-compile-remove(hide-attendee-name) */ role,
+          undefined || isHideAttendeeNamesEnabled,
```
Comment (mgamis-msft) [inline packages/calling-component-bindings/src/participantListSelector.ts:162]: Can remove the `undefined`'s here

**Review by mgamis-msft — APPROVED** [review]

File: packages/calling-component-bindings/src/participantListSelector.ts (hunk 1 above)
Comment (dmceachernmsft) [reply packages/calling-component-bindings/src/participantListSelector.ts:162]: Ah not my feature so wasn't sure. will look at that

File: packages/calling-component-bindings/src/utils/videoGalleryUtils.ts [hunk 2]
```diff
@@ -62,7 +58,6 @@ export const _videoGalleryRemoteParticipantsMemo: _VideoGalleryRemoteParticipant
         .map((participant: RemoteParticipantState) => {
           const state = _isRingingPSTNParticipant(participant);
           let displayName = participant.displayName;
```
Comment (JamesBurnside) [inline packages/calling-component-bindings/src/utils/videoGalleryUtils.ts:60]: remove line here also

File: packages/react-components/src/components/ParticipantList.tsx [hunk 3]
```diff
@@ -140,14 +140,12 @@ const onRenderParticipantDefault = (
 
   let displayName = participant.displayName;
```
Comment (JamesBurnside) [inline packages/react-components/src/components/ParticipantList.tsx:141]: remove line

File: packages/react-components/src/components/ParticipantList.tsx [hunk 4]
```diff
@@ -140,14 +140,12 @@ const onRenderParticipantDefault = (
...
 
-  /* @conditional-compile-remove(hide-attendee-name) */
   const formatDisplayName = (): string | undefined => {
```
Comment (JamesBurnside) [inline packages/react-components/src/components/ParticipantList.tsx:143]: remove (seemingly) trampoline function

File: packages/react-components/src/components/RemoteVideoTile.tsx [hunk 5]
```diff
@@ -189,10 +188,8 @@ export const _RemoteVideoTile = React.memo(
     );
 
     let displayName = remoteParticipant.displayName || strings.displayNamePlaceholder;
-    /* @conditional-compile-remove(hide-attendee-name) */
     const attendeeRoleString = props.strings?.attendeeRole;
 
-    /* @conditional-compile-remove(hide-attendee-name) */
     const formatDisplayName = (): string => {
```
Comment (JamesBurnside) [inline packages/react-components/src/components/RemoteVideoTile.tsx:193]: remove trampoline function

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**